<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameMainDialog</class>
 <widget class="QDialog" name="GameMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameMainDialog</string>
  </property>
  <widget class="QPushButton" name="StartSingleplayer">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>281</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Singleplayer Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StartEditor">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>281</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Editor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseGame">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>281</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Close Game</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
